src/editor/ImageViewer.js

```diff
@@ -153,7 +151,7 @@ define(function (require, exports, module) {
         this._naturalHeight = e.currentTarget.naturalHeight;
 
         var extension = FileUtils.getFileExtension(this.file.fullPath);
-        var dimensionString = this._naturalWidth + " &times; " + this._naturalHeight + " " + Strings.UNIT_PIXELS;
+        var dimensionString = this._naturalWidth + " (width) &times; " + this._naturalHeight + " (height) " + Strings.UNIT_PIXELS;
```

Comment:
These are going to have to get changed to localized strings that use the `Strings` module.  cc @gideonthomas who can guide you on how to make such a change.